robyn/import_vars.py

```diff
@@ -0,0 +1,27 @@
+import os 
```

Comment:
Maybe name this file something like `environment_populator` or `env_populator`? Or something similar according to you.

As a good practice is to keep the namespace or filename in accordance to what it does. And I assume the purpose of this file is to have helper functions for environment population.